src/main/resources/db/migration/V1__init_tables.sql

```diff
@@ -0,0 +1,21 @@
+create table stillinger(
+	id varchar(128) not null primary key,
+    kilde varchar(200) not null, -- Skal vi bruke NAV her, eller virkelig kilde???
+    status varchar(100) not null,
+    opprettet_ts timestamp not null,
+    sist_endret_ts timestamp not null,
+    varer_til_ts timestamp null,
+	json_stilling clob not null
```

Comment:
Tror vi burde ha xml også i database. Dersom man på sikt også skal fungere som proxy for 3. parter, så må vi kunne lagre xml fra dem. Hvis vi lagrer ferdig tygd xml fra oss, så vil vi slippe å gjøre noe selv annet enn å la dem pushe til oss